### ISSUE TITLE
Version 2.6.1(e): When completing evaluations, the head judge and/or admin marks a placement for an entry. This information is not saved. This will be a big problem when preparing for BOS, and it's also not saving Mini-BOS information. 

### ISSUE THREAD

**Issue by kmeintsma** [geoffhumphrey/brewcompetitiononlineentry#1466]
**Verify the Following**
Reporting About the Latest Version? Yes or No
*If you are not running the most current version in your environment, please update prior to reporting your bug. Your issue may have already been addressed.*

Searched Issue Reports for Bug? Yes or No
*Please search open *and* closed issues for issues similar to yours. Your bug may already have a posted solution.*

Reviewed the Good To Know Issue Reports? Yes or No
*Please review the Issue reports that are labeled Good to Know. Find one and click on the label to view them all.*

**BCOE&M Info**
Version: 
Installation URL: 
Hosted Installation? Yes or No

**Describe the Bug**

**To Reproduce**

**Expected Behavior**

**Screenshots**

**Server Environment**
- PHP Version:
- MySQL Version: 
- Other Info: [if applicable. Apache, nginx, etc.]

**Local Environment**
 - OS:
 - Browser: 
 - Browser Version:

**Additional Context**

**Comment by kmeintsma** [comment]
Additional environment information:

    PHP Version – 7.4.33;
    MariaDB Version – 5.5.5-10.6.15-MariaDB-1:10.6.15+maria~ubu2204
    
    Does not matter what browser is running - Firefox, Chrome, Edge, Safari are all the same.

**Comment by geoffhumphrey** [comment]
Thanks for the info. Tracked down the problem. Change [line 986 in /eval/dashboard.eval.php](https://github.com/geoffhumphrey/brewcompetitiononlineentry/blob/cc1eab3bafbd7966d1db3a94819c6395faf16555/eval/dashboard.eval.php#L986) to this:
```
<script src="<?php echo $base_url;?>js_includes/admin_ajax.min.js"></script>
```
Not sure why that was changed, but I tested and adding the above code will alleviate the issue.
